be/src/agent/agent_task.cpp

```diff
@@ -223,7 +223,7 @@ void run_create_tablet_task(const std::shared_ptr<CreateTabletAgentTaskRequest>&
                      << ", signature: " << agent_task_req->signature;
         status_code = TStatusCode::RUNTIME_ERROR;
         if (tablet_type == TTabletType::TABLET_TYPE_LAKE) {
-            error_msgs.emplace_back("create tablet failed");
+            error_msgs.emplace_back(create_status.to_string_for_client());
```

Comment:
Why not keep the "Fail to create tablet: " to let users know how this error message is generated?

Reply:
The old error message contained "failed to create tablet" twice.
![image](https://github.com/StarRocks/starrocks/assets/145319350/18d44874-5340-4e16-afc5-f43f49cce164)

This "Fail to create tablet" is not necessary. It is replaced by a more detailed error message. The other "fail to create tablet:" is added in the following fucntion:
![image](https://github.com/StarRocks/starrocks/assets/145319350/1fd92346-0cf3-4af0-9f17-9798ed9cfedf)

With this commit, the error message to client now is like:
![image](https://github.com/StarRocks/starrocks/assets/145319350/9ab22205-4617-47d3-91c5-db758eaa3fa8)